src/main/java/com/hotel/api/book/model/request/UserInfo.java

```diff
@@ -0,0 +1,10 @@
+package com.hotel.api.book.model.request;
+import lombok.Data;
+@Data
```

Comment:
Just give a space between the import and annotation
This is a model you have to create getters and setter . so need to give following annotations
@setter
@getter
@AllArgsConstructor
@NoArgsConstructor
@tostring
@builder(toBuilder = true)